stacks_queues/queue.py

```diff
@@ -15,47 +15,71 @@ def __init__(self):
         self.front = -1
         self.rear = -1
         self.size = 0
-      
 
     def enqueue(self, element):
         """ Adds an element to the Queue
             Raises a QueueFullException if all elements
             In the store are occupied
             returns None
         """
-        pass
+        if self.size == INITIAL_QUEUE_SIZE:
```

Comment:
This absolutely works! However, our general preference code style-wise is to use our attributes over global constants 
```suggestion
        if self.size == self.buffer_size:
```